src/features/defi/components/Overview/UnderlyingAssetsMenu.tsx

```diff
@@ -0,0 +1,51 @@
+import { ChevronDownIcon } from '@chakra-ui/icons'
+import { Box, Flex, Popover, PopoverBody, PopoverContent, PopoverTrigger } from '@chakra-ui/react'
+import { Amount } from 'components/Amount/Amount'
+import { AssetIcon } from 'components/AssetIcon'
+
+import { PairIcons } from '../PairIcons/PairIcons'
+import type { AssetWithBalance } from './Overview'
+import { UnderlyingAssetsTags } from './UnderlyingAssetsTags'
+
+type UnderlyingAssetsMenuProps = {
+  underlyingAsset: AssetWithBalance
```

Comment:
`asset` might make more sense? Technically the LP asset is the asset, and the two assets it represents are in fact the underlying ones

Reply:
As with the types on `OverviewProps`, `underlyingAsset` and `underlyingAssets` side-by-side is confusing.

I think changing `underlyingAsset` to `lpAsset` would be more appropriate here.

Reply:
That sounds good to me! Agreed taking a step back and looking into this, it does look confusing side by side

32e0357ecb92199ceb456d245b8578163a11b590